CMake/SlicerMacroExtractRepositoryInfo.cmake

```diff
@@ -67,12 +67,13 @@ macro(SlicerMacroExtractRepositoryInfo)
   set(${wc_info_prefix}_WC_TYPE local)
   set(${wc_info_prefix}_WC_URL "NA")
   set(${wc_info_prefix}_WC_ROOT "NA")
+  set(${wc_info_prefix}_WC_REVISION "NA")
```

Comment:
Unconditionally setting `..._WC_REVISION` would prevent specifying the revision manually.